packages/odyssey/src/scss/components/_tag.scss

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 .ods-tag {
+  display: inline-block;
```

Comment:
@arnoldsandoval-okta looks like this is necessary to ensure the same display we have in the docs today. Probably previous oversight on my part.